Assets/Scripts/Device/DiscordControls.cs

```diff
@@ -42,28 +42,28 @@ public class DiscordControls {
         if (details != "") {
             rpDetails = details;
         }
-		
-		if (time != 0) {
-			rpTime = ((int)(System.DateTime.UtcNow - epochStart).TotalSeconds) + time;
-		} else {
-			rpTime = 0;
-		}
-		
-		
-		var activityManager = discord.GetActivityManager();
+        
+        if (time != 0) {
+            rpTime = ((int)(System.DateTime.UtcNow - epochStart).TotalSeconds) + time;
+        } else {
+            rpTime = 0;
+        }
+        
+        
+        var activityManager = discord.GetActivityManager();
 
         var activity = new Discord.Activity
```

Comment:
We usually try to keep curly braces on the same line as the block that uses them.
I'm only making one comment about it, but I hope you update all occurences of this problem.
```suggestion
        var activity = new Discord.Activity {
```